build-tools/src/integTest/groovy/org/elasticsearch/gradle/plugin/PluginBuildPluginFuncTest.groovy

```diff
@@ -157,7 +157,11 @@ class PluginBuildPluginFuncTest extends AbstractGradleFuncTest {
     }
 
     Map<String, String> getPluginProperties() {
-        Path propsFile = file("build/generated-descriptor/plugin-descriptor.properties").toPath();
```

Comment:
That seems not required anymore as we split the tests of Stable Plugin Build Plugin and and Build Plugin.